src/Microsoft.Health.Dicom.Blob/Features/Storage/BlobMetadataStore.cs

```diff
@@ -148,13 +144,7 @@ public Task<DicomDataset> GetInstanceMetadataAsync(VersionedInstanceIdentifier v
                         versionedInstanceIdentifier);
                     break;
                 case JsonException or NotSupportedException:
-                    _telemetryClient
-                        .GetMetric(
-                            JsonDeserializationException,
-                            JsonDeserializationExceptionTypeDimension)
-                        .TrackValue(
-                            1,
-                            ex.GetType().FullName);
+                    _blobStoreMeter.JsonDeserializationException.Add(1, new[] { new KeyValuePair<string, object>("JsonDeserializationExceptionTypeDimension", ex.GetType().FullName) });
```

Comment:
Presumably this is not part of a store operation, is it? I assume this is a retrieval using the WADO APIs?

Reply:
Moved it to `BlobRetrieveMeter`